docs/g3tsmurf.rst

```diff
@@ -7,16 +7,70 @@ G3tSMuRF
 
 G3tSmurf currently consists of two parts.
 
-1. A databasing setup for tracking available SMuRF data saved using the SO
+1. A database setup for tracking available SMuRF data saved using the SO
 software stack. 
 
-2. A mechanism for loading that data.
+2. Mechanisms for loading that data.
 
-Eventually the databasing part will be separated out and migrated to work within
-the Context system and the loading data functions will be altered to load data
-through a Context object. 
+The G3tSmurf database can be used to load data through `load_data` or
+`load_file`. The only real difference between these (currently) is that
+`load_data` accepts a start and stop time while `load_file` accepts a list of
+paths.
 
+A tutorial with examples of data loading is avaliable
+`here
+<https://github.com/simonsobs/pwg-tutorials/blob/master/sotodlib_G3tSMuRF_Tutorial_20210415.ipynb>`_.
+The description of the databases are partially out of date.
 
+Database Updating
+-----------------
+Here is an example database update script::
+    
+    from sotodlib.io.load_smurf import (G3tSmurf, Observations)
+
+    SMURF = G3tSmurf( archive_path='/data/timestreams/',
+                      meta_path='/data/smurf/', 
+                      db_path='/path/to/things/g3t_smurf_db.db')
+    SMURF.index_archive()
+    SMURF.index_metadata()
+
+    ## Check recent observations for updates
+    ## Accounts for partial data transfers
+    session = SMURF.Session()
+    new_obs = session.query(Observations)
+    new_obs = new_obs.filter( Observations.start >= dt.datetime.now()-dt.timedelta(days=1))
+    new_obs = new_obs.all()
+    for obs in new_obs:
+        SMURF.update_observation_files(obs, session)
+    session.close()
+
+Usage with Context
+------------------
+The G3tSmurf database can now be used with the larger `sotodlib` Context system.
+In this setup, the main G3tSmurf database is both the ObsFileDb and the ObsDb.
+The DetDb needs to be created using the `dump_DetDb(SMURF, detdb_file)`. This
+function call can be added to the standard database update script to keep it up
+to date. 
+
+Example context yaml file::
+
+    tags:
+    g3tsmurf_dir: '/path/to/things/smurf_context'
+
+    obsfiledb: '{g3tsmurf_dir}/g3t_smurf_db.db'
+    obsdb: '{g3tsmurf_dir}/g3t_smurf_db.db'
+    detdb: '{g3tsmurf_dir}/detdb.db'
+
+    imports:
+    - sotodlib.io.load_smurf
+
+    obs_loader_type: 'g3tsmurf'
+
+    #metadata:
+  
+
```

Comment:
It looks (in the code) like obs_id corresponds to G3 filenames... please mention that briefly in this documentation (or direct the user to the docstrings that elucidate the mapping).

What other info is made available through ObsDb?  Can I select observations based on the slot_id (for example)?

Reply:
Went ahead and added more docstrings. 

slot_id isn't technically something we'll be able to select on, it's going to line up exactly with tunesets/detsets.